### PR TITLE
fix: re-add action for building and publishing docs []

### PR DESCRIPTION
**Pull request by jjolton-contentful**
https://contentful.github.io/node-apps-toolkit/

### CODE REVIEW

File: .github/workflows/pages-build-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+
+      - name: Install Dependencies
+        run: npm install
```
Comment (whitelisab) [inline .github/workflows/pages-build-deployment.yml:22]: Just curious about using `npm install` here vs. `npm ci`?

File: package-lock.json
Comment (whitelisab) [inline package-lock.json:1]: Was the `package-lock.json` just out of sync? Just a sanity check that there didn't need to be any changes to the `package.json` file with all these updates.

**Review by whitelisab — DISMISSED** [review]
Nice to have those docs! 🚀 Just added some small non-blocking questions, mostly for my own curiosity 😄

File: .github/workflows/pages-build-deployment.yml (hunk 1 above)
Comment (jjolton-contentful) [reply .github/workflows/pages-build-deployment.yml]: good call, updated :+1:

File: package-lock.json
Comment (jjolton-contentful) [reply package-lock.json:1]: good eye, see reply in other thread 🧵

**Review by t-col — APPROVED** [review]

**Pull request merged**